Application/Source/Models/Content.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import RxCocoa
+
+protocol Content {
+    associatedtype CategoryType
+
+    var name: BehaviorRelay<String> { get }
+    var category: CategoryType { get }
+}
+
+/**
+ It's not a side project if you can't have some fun, right?
+ - [Original.](https://www.youtube.com/watch?v=amONEHAhLHY)
+ - [Crispy.](https://www.youtube.com/watch?v=1BC1G33-fNY)
+ */
+enum Food: Content {
+
+    case beans, greens, potatoes, tomatoes
+
+    enum Category: Int {
+        case fruit, tuber, vegetable, legume
+    }
+
+    var name: BehaviorRelay<String> {
```

Comment:
Does this need to return a `BehaviorRelay<String>`? It seems like these values would remain static. 

Should this should return a `String` instead?